### PR TITLE
[Zip][Inno] New package: HxD 2.5

### PR DESCRIPTION
**Pull request by hikmet517**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96195)

### CODE REVIEW

File: manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=MDSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MHNexus.HxD
+PackageVersion: 2.5.0.0
```
Comment (Trenly) [inline manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.installer.yaml:5]:
```suggestion
PackageVersion: 2.5
```
Should match Control Panel

File: manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=MDSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MHNexus.HxD
+PackageVersion: 2.5.0.0
```
Comment (Trenly) [inline manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.5
```
Should match Control Panel

File: manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=MDSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: MHNexus.HxD
+PackageVersion: 2.5.0.0
```
Comment (Trenly) [inline manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.yaml:5]:
```suggestion
PackageVersion: 2.5
```
Should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]
```
--> Comparing ARP Entries

DisplayName        DisplayVersion Publisher ProductCode
-----------        -------------- --------- -----------
HxD Hex Editor 2.5 2.5            Maël Hörz HxD_is1
```

**Review by Trenly — APPROVED** [review]

**Comment by hikmet517** [comment]
> Hello @wingetbot,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

All the URLs:
InstallerUrl: https://mh-nexus.de/downloads/HxDSetup.zip
PublisherUrl: https://mh-nexus.de/
PublisherSupportUrl: https://forum.mh-nexus.de/viewforum.php?f=7
PackageUrl: https://mh-nexus.de/en/hxd/
LicenseUrl: https://mh-nexus.de/en/hxd/license.php
CopyrightUrl: https://mh-nexus.de/en/hxd/license.php

I don't see anything wrong, all the links are working.

**Pull request merged**